TheFourFifths/consus-client
# Check in UI

### Summary

1) Added check in functionality to the cart's item input
2) Added the auto-selects for the omnibar and cart input

### Checklist


Finally, is it ready to be reviewed and merged: 🍏 

### How to Review

1. Run this client and the server from https://github.com/TheFourFifths/consus/tree/check-in-ui-TFF-70
2. enter a valid student id and check out an item
3. Return the item by entering the item address again

### Links

- [TFF-70](https://msoese.atlassian.net/browse/TFF-70)
